cvrf2csaf/section_handlers/document_notes.py

```diff
@@ -3,11 +3,52 @@
 
 
 class DocumentNotes(SectionHandler):
-    def __init__(self):
-        super().__init__()
+    def __init__(self, config):
+        super().__init__(config=config)
+
+        self.enum_categories = {
+            'description',
+            'details',
+            'faq',
+            'general',
+            'legal_disclaimer',
+            'other',
+            'summary'
+        }
+
+    def _process_mandatory_and_optional(self, root_element):
+        notes = []
+
+        for elem_note in root_element.Note:
+
+            # mandatory
+            new_note = dict(
+                text=elem_note.text,
+                category=elem_note.get('Type'),
```

Comment:
This inserts the text directly without converting into the correct format.

Reply:
Replace whitespace with underscore in value + turn into lower case.

Reply:
Fixed in a5658c2